ansible-collection-redhatci-ocp.spec

```diff
@@ -28,7 +28,7 @@ Requires: python3-jmespath
 Requires: python3-netaddr
 Requires: python3-pyyaml
 Requires: skopeo
-Conflicts: dci-openshift-agent < 1.1.0
+Conflicts: dci-openshift-agent < 1.6.0
```

Comment:
This is not needed because older versions of dci-openshift-agent are not affected with this change. The requirement is satisfied in the dci-openshift-agent by using >= 1.1.0 (although there is never a 1.1.0 version).